src/components/snapshot/overview/ManageWorkspacesModal.tsx

```diff
@@ -104,52 +73,67 @@ export class ManageWorkspacesModal extends React.PureComponent<
   }
 
   handleClickOpen = () => {
-    this.setState({
-      open: true,
-    });
+    this.setState({ open: true });
   };
 
   handleClose = () => {
     this.setState({ open: false });
   };
 
   render() {
-    const { classes, modalText, entries, removeWorkspace, isLoading } = this.props;
+    const { modalText, entries, removeWorkspace, isLoading } = this.props;
     const { open } = this.state;
-    const button = (
-      <Button
-        className={classes.openButton}
-        aria-label={modalText}
-        onClick={this.handleClickOpen}
-        disableFocusRipple={true}
-        disableRipple={true}
-      >
-        <i className={`${classes.openButtonIcon} fa-solid fa-pen-circle`} />
-        {modalText}
-      </Button>
-    );
 
     return (
-      <span>
-        {button}
+      <Box component="span">
+        <OpenButton
+          aria-label={modalText}
+          onClick={this.handleClickOpen}
+          disableFocusRipple
+          disableRipple
+        >
+          <Box
+            component="i"
+            className="fa-solid fa-pen-circle"
+            sx={{ marginRight: '5px', top: (theme) => theme.spacing(1) }}
+          />
+          {modalText}
+        </OpenButton>
         <Dialog
           fullWidth
           maxWidth="md"
           onClose={this.handleClose}
           aria-labelledby="customized-dialog-title"
           open={open}
         >
-          <DialogTitle className={classes.dialogTitle} id="customized-dialog-title">
+          <DialogTitle
+            id="customized-dialog-title"
+            sx={{
+              borderBottom: (theme) => `1px solid ${theme.palette.divider}`,
+              margin: 0,
+              padding: (theme) => theme.spacing(2),
+            }}
+          >
             {modalText}
             <IconButton
               aria-label="Close"
-              className={classes.closeButton}
               onClick={this.handleClose}
+              sx={{
+                position: 'absolute',
+                right: (theme) => theme.spacing(1),
+                top: (theme) => theme.spacing(1),
+                color: (theme) => theme.palette.grey[500],
```

Comment:
```suggestion
                right: 1,
                top: 1,
                color: 'grey[500]',
```
Same for palette: https://mui.com/system/getting-started/the-sx-prop/#spacing